### PR TITLE
Votecomponent

### PR DESCRIPTION
**Pull request by guenk**

### CODE REVIEW

File: frontend/src/components/Vote/Vote.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import "./style.scss";
+
+export default function Vote() {
+  return (
+    <div className="vote">
+      <label>
+        <h2 className="proptitle">Première proposition</h2>
+        <textarea id="fpro1" name="fpro" rows="15" />
+        <button className="vote-button" type="button">
+          Voter
```
Comment (Sax593) [inline frontend/src/components/Vote/Vote.jsx]: Change Voter by Add

File: frontend/src/components/Vote/Vote.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import "./style.scss";
+
+export default function Vote() {
+  return (
+    <div className="vote">
+      <label>
+        <h2 className="proptitle">Première proposition</h2>
+        <textarea id="fpro1" name="fpro" rows="15" />
```
Comment (Sax593) [inline frontend/src/components/Vote/Vote.jsx]: Change textarea by input. Choice tabel in DB accepte only varchar (128)

**Review by Sax593 — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -18,5 +18,8 @@
   "devDependencies": {
     "concurrently": "^7.0.0",
     "husky": "^7.0.0"
+  },
+  "dependencies": {
+    "chart.js": "^4.1.2"
```
Comment (Sax593) [inline package.json]: Please install in front

**Review by Sax593 — CHANGES_REQUESTED** [review]

**Review by Sax593 — DISMISSED** [review]

File: frontend/src/pages/SuggestForm/style.scss [hunk 4]
```diff
@@ -24,10 +24,10 @@ label {
...
 }
 
-.inputText,
-.textArea {
-  max-width: 20vw;
-}
+// input,
+// textarea {
+//   max-width: 20vw;
+// }
```
Comment (LoicBrassart) [inline frontend/src/pages/SuggestForm/style.scss]: Dead code

File: frontend/src/components/Vote/style.scss [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+  transition: 0.2s;
+}
+
+.vote-button:hover:after {
+  top: 0px;
+  left: 0px;
+}
+
+.pstyle {
+  width: 100%;
```
Comment (LoicBrassart) [inline frontend/src/components/Vote/style.scss]: Useless in most cases, that's the default width

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Minor Evils

File: frontend/src/components/Vote/style.scss (hunk 5 above)
Comment (guenk) [reply frontend/src/components/Vote/style.scss]: fix

File: frontend/src/pages/SuggestForm/style.scss (hunk 4 above)
Comment (guenk) [reply frontend/src/pages/SuggestForm/style.scss]: fix

**Review by Sax593 — DISMISSED** [review]

**Review by LoicBrassart — DISMISSED** [review]
sne

**Review by Sax593 — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**